apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/WaRuleMetadatum.java

```diff
@@ -79,4 +81,22 @@ public class WaRuleMetadatum {
     @Column(name = "target_type", length = 50)
     private String targetType;
 
+    WaRuleMetadatum (WaTemplate page ,PageContentCommand.AddRule command){
+        this.waTemplateUid = page;
+        this.ruleCd= command.ruleCd();
+        this.errMsgTxt= command.errMsgTxt();
+        this.javascriptFunction= command.javascriptFunction();
+        this.javascriptFunctionNm= command.javascriptFunctionNm();
+        this.recordStatusCd= command.recordStatusCd();
+        added(command);
+    }
+    private void added(PageContentCommand command) {
+        this.addTime = command.requestedOn();
+        this.addUserId = command.userId();
+        this.lastChgTime = command.requestedOn();
+        this.lastChgUserId = command.userId();
+        this.recordStatusTime = command.requestedOn();
+
+    }
+
```

Comment:
Issues: don't think this would be needed as the WaTemplate will call `WaRuleMetadata` instead of `WaRuleMetadatum`

Reply:
Pls see my comment above